src/app/test-record/test-delete-reason-modal/test-delete-reason-modal.component.html

```diff
@@ -0,0 +1,28 @@
+<div mat-dialog-content>
+  <h1 class="govuk-heading-m enter-reason-for-cha">Enter reason for deleting test record</h1>
+  <textarea
+    class="govuk-textarea"
+    id="reasonForDelete"
+    rows="8"
+    [(ngModel)]="modalContent.payload"
+    aria-describedby="reasonForDelete"
+  >
+  </textarea>
+</div>
+
+<div class="govuk-inset-text">
+  <h2 class="govuk-heading-m">Any linked tests will also be deleted.</h2>
+</div>
+
+<button
+  class="govuk-button govuk-button--warning"

```

Comment:
do we need an id as well on that?
We have to check with @DeepikaSinghD @dpanzaru-deloitte